src/feditest/testrun.py

```diff
@@ -213,7 +216,7 @@ def run(self):
 
         run_sessions: list[TestRunSession] = []
 
-        for i in range(0, len(self._plan.sessions)):
+        for i in range(0, len(self._plan.sessions)): # pylint: disable=consider-using-enumerate
```

Comment:
I'm curious why you are disabling this? (E.g., `for i, plan_session in enumerate(self._plan.sessions):`)

Reply:
Trying to get the number of pylint reports down. Again there may be better ways...

Reply:
Yes, the code in the example I gave is what pytest is recommending.